### PR TITLE
feat(CORE/docker): switch to `docker compose` and check the version

### PR DESCRIPTION
**Pull request by Yehonal**
## Changes Proposed:
-  Switch from `docker-compose` to `docker compose` that is integrated with docker now
- Implemented a check to make sure that the docker compose version is high enough

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by MikaMauger** [comment]
LGTM, tested on Debian 11 (Bullseye)

```
$ docker --version
Docker version 20.10.19, build d85ef84
```

```
$ docker compose version
Docker Compose version v2.11.2
```

**Comment by BarbzYHOOL** [comment]
"feat(CORE/docker)" --> "chore(docker)" @Yehonal

**Comment by luokeli59** [comment]
# OS:
    ubuntu 20.04 server

# docker version
    Client: Docker Engine - Community
     Version:           20.10.21

# docker-compose version
    Docker Compose version v2.12.2

# ./acore.sh docker build
    ./acore.sh docker dev:biild
    NOTICE: file </root/azerothcore-wotlk/conf/config.sh> not found, we use default configuration only.
    Deno version check:
    Version 1.26.0 is already installed
    docker: 'compose' is not a docker command.
    See 'docker --help'
    ERROR: Your docker-compose version (not installed?) must be higher or equal to 2.0.0. Please install the new version of docker-compose and try again
